src/models/user.js

```diff
@@ -55,6 +55,12 @@ export default (sequelize, DataTypes) => {
       },
     },
     role: DataTypes.ENUM(roles),
+    fullName: {
+      type: DataTypes.VIRTUAL,
+      get() {
+        return `${this.name}, ${this.role.split(' ').map((word) => word[0]).join('')}`;
```

Comment:
This is causing a 500 error when updating users with a null role. Please deal with a missing role cleanly.